docs/Tools_Equipment/Banned Tools.md

```diff
@@ -0,0 +1,12 @@
+# Banned Tools & Equipment
+
+This list is includes tools deemed too dangerous and unsuitable for the Hackspace.  Usage of these tools on or near the vicinity of the space is strictly forbidden.
+
+This list is not definitive and tools will be added when required.  If you are unsure whether a tool can be used in the space please contact the board for clarification.
+
+### **Banned Tools:**
+- Chainsaws
+- Dado blades
+- Flamethrowers
```

Comment:
This reads like a mix of sensible and frivolous advice. I don't think jokes belong in safety documentation.

Reply:
So I'm assuming you're referring to the flamethrowers, it wasn't intended as a joke.  I wanted to put open flame tools but since we allow blow torches I wasn't sure what to put.

Reply:
Ah OK, point about it being frivolous retracted, then... but I'm not sure what you're referring to by flamethrowers.

Reply:
Probably improvised flamethrowers such as colin fruze.

I think in the case of Dado blades i think you can only get ones that match with a given type of table saw which is probably sold in America, so there's not much of a risk of someone being able to find one that would actually fit our saw anyway.

Chainsaws was something that came up in the chat recently with Alex in relation to chopping up bits of wood for the wood lathe.